test/setup/main.tf

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_id" "project" {
+  byte_length = 4
+}
+
+locals {
+  project_name = "ci-event-function-${random_id.project.hex}"
+}
+
+
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 3.0"
+
+  name                = local.project_name
+  random_project_id   = false
+  auto_create_network = true
```

Comment:
This should be `false`; we don't want to rely on the default network.

Reply:
this is the reason I've added true, please advice how to address this it proper way without default networks

Reply:
Apologies; that's a fair point. I've created https://github.com/terraform-google-modules/terraform-google-event-function/issues/33 to track this change separately.